maven/plugins/com.liferay.ide.maven.core/src/com/liferay/ide/maven/core/pref/MavenArchetypePreferenceInitializer.java

```diff
@@ -72,7 +72,7 @@ public void initializeDefaultPreferences() {
 
 		defaultPrefs.put(
 			LiferayMavenCore.PREF_ARCHETYPE_PROJECT_TEMPLATE_WORKSPACE,
-			"com.liferay:com.liferay.project.templates.workspace:1.0.4");
+			"com.liferay:com.liferay.project.templates.workspace:1.0.7");
```

Comment:
this should be done via blade now and we should no longer use it from the archetype.  @JoyeLuo  can you add support to the `blade init` command to use maven?  then we can use blade for creating a maven workspace.  Thanks!

Reply:
Then we can remove this template all together.

Reply:
cc @jtydhr88

Reply:
hey @gamerson can you merge it firstly as It causes some bugs as the old template use old blade.support lib, I will ask @JoyeLuo work on the Maven support today. thanks